lib/nvme/nvme_pcie.c

```diff
@@ -133,8 +133,8 @@ struct nvme_pcie_qpair {
 	/* Completion queue */
 	struct spdk_nvme_cpl *cpl;
 
-	LIST_HEAD(, nvme_tracker) free_tr;
-	LIST_HEAD(, nvme_tracker) outstanding_tr;
+	TAILQ_HEAD(, nvme_tracker) free_tr;
```

Comment:
I'm not opposed to changing this LIST to TAILQ if it is needed, but it would be nice if this were a separate patch before the timeout change to make review easier.